flybywiresim/aircraft
# fix: wheels and chocks element clash

Fixes #8223

## Summary of Changes

Hotfix for chocks and Cones not overlapping with aft door

![image](https://github.com/flybywiresim/aircraft/assets/15316958/61307389-5500-4f2d-9b29-8da8fca5f271)


## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

1. Ensure that Chocks/Cones setting is enabled in the EFB Settings > Sim Options
![image](https://github.com/flybywiresim/aircraft/assets/15316958/2a5c34ae-d047-42a1-a6df-b0d0cba6ed94)
2. Check that Cone/Chocks elements do not overlap on the EFB Ground Services Page
3. Disable Chocks/Cones Setting in the EFB settings > Sim Options page
4. Check that Cone/Chocks elements do not appear on the EFB Ground Services Page


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
